site/AntDesign.Docs/Demos/Components/DatePicker/doc/index.en-US.md

```diff
@@ -48,6 +48,7 @@ The following APIs are shared by DatePicker, YearPicker, MonthPicker, RangePicke
 | style | to customize the style of the input box | CSSProperties | {} |  |
 | onOpenChange | a callback function, can be executed whether the popup calendar is popped up or closed | function(open) | - |  |
 | onPanelChange | callback when picker panel mode is changed | function(value, mode) | - |  |
+| OnClearClick | callback when clear button is clicked | Action | - |  |
```

Comment:
`OnClearClick` or `OnClickClear` ? I prefer `OnClearClick`.

Reply:
Of course `OnClearClick` 🤦🏻